app/templates/partials/start-application-button.html

```diff
@@ -1,14 +1,15 @@
 {%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
 
 {% macro startApplicationButton(form, url, applications) %}
```

Comment:
Do we need to pass in the `account_id, fund_id & round_id` here?

Reply:
no need to pass these variables 👍🏾